stdlib/filepath/src/main/ballerina/filepath/file_path.bal

```diff
@@ -0,0 +1,460 @@
+// Copyright (c) 2019 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/io;
+import ballerina/log;
+import ballerina/system;
+
+boolean IS_WINDOWS = system:getEnv("OS") != "";
+string PATH_SEPARATOR = IS_WINDOWS ? "\\" : "/";
+string PATH_LIST_SEPARATOR = IS_WINDOWS ? ";" : ":";
```

Comment:
Can we use `private const` fields here?

Reply:
These values are derived at runtime So we can't make them as compile time constants.

Reply:
can we make them private?

Reply:
If they are not constants why do we have them in capital? Also can we make them private as private fields are now supported.